pkg/api/v1/resource_helpers.go

```diff
@@ -17,10 +17,14 @@ limitations under the License.
 package v1
 
 import (
+	"fmt"
+	"math"
+	"strconv"
 	"time"
 
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
Sad to see we still need to import `pkg/api` because we need to access the Scheme, but it looks like we have to live with it.